install.sh

```diff
@@ -1,18 +1,26 @@
 #!/bin/bash
 
-gpg_verify_key ()
+_gpg ()
 {
-    gpg --keyid-format long <"$1" | grep "$2"
+    gpg --no-default-keyring --keyring "${jm_deps}/keyring.gpg" "$@"
 }
 
 gpg_add_to_keyring ()
 {
-    gpg --dearmor <"$1" >>"${jm_deps}/keyring.gpg"
+    if _gpg --list-keys "$1"; then
+        return 0
+    fi
+    for keyserv in 'pgp.mit.edu' 'keys.gnupg.net'; do
+        if _gpg --keyserver "${keyserv}" --recv-keys "$1"; then
```

Comment:
Since I suggested `--keyserver` in IRC I feel partially responsible to answer this.

Also from the man page:

> --recv-keys key IDs
              Import  the  keys with the given key IDs from a keyserver. Option --keyserver
              must be used to give the name of this keyserver.

Even though it's deprecated, I get no complaints on stdout by adding `--keyserver` from `gpg` version `2.1.18`. Therefore it seems like it might be good to keep on for users of older versions, or those that don't have it set in `dirmngr`.